docs/source/tutorial/dependency-management.md

```diff
@@ -0,0 +1,93 @@
+
+## Changelog
+
+All Availity open-source projects maintain a changelog. Each project maintains it's own changelog. The changelog can be found either in the `releases` link on GitHub or as a `CHANGELOG.md` at the root of the project.
+
+### Releases Examples
+
+-   <https://github.com/Availity/availity-workflow/releases>
+-   <https://github.com/Availity/sdk-js/releases>
+-   <https://github.com/Availity/availity-uikit/releases>
+-   <https://github.com/Availity/sdk-js/releases>
+
+### Changelog.md Examples
+
+-   <https://github.com/Availity/sdk-js/blob/master/CHANGELOG.md>
+-   <https://github.com/Availity/availity-workflow/blob/master/CHANGELOG.md>
+-   <https://github.com/Availity/availity-react/blob/master/CHANGELOG.md>
+
+## Tools
+
+It is up to the development team to decide if they want or need to upgrade any library in their stack. There are some tools available that help teams know if new releases of components have been published for use:
+
+### `npm outdated`
+
+```bash
+❯ npm outdated --depth=0
+Package                                 Current            Wanted        Latest  Location
+@uirouter/angularjs                       1.0.4             1.0.4        1.0.15  availity-toolkit > availity-angular
+bootstrap-datepicker                      1.7.0             1.7.0         1.7.1  availity-toolkit > availity-angular
+awesome-bootstrap-checkbox                0.3.7             0.3.7         1.0.0  availity-toolkit > availity-uikit
+bootstrap                                 3.3.7             3.3.7         4.0.0  availity-toolkit > availity-uikit
+```
+
+### `npm-check`
+
+<https://www.npmjs.com/package/npm-check>
+
+```bash
+ npm-check -u
+? Choose which packages to update.
+
+ Major Update Potentially breaking API changes. Use caution.
+❯◉ bootstrap           3.3.7             ❯  4.0.0  <https://getbootstrap.com>
+ ◯ select2             3.5.2-browserify  ❯  4.0.5  <https://select2.org>
+ ◯ lint-staged devDep  6.1.1             ❯  7.0.0  <https://github.com/okonet/lint-staged#readme>
+
+ Space to select. Enter to start upgrading. Control-C to cancel.
+```
+
+## FAQ
+
+### What happens when teams upgrade a library and the project/build breaks?
+
+-   Check the library changelog for BREAKING CHANGE announcements or recent fixes
+-   Check the offending library issue board and see if tickets have been
+    opened or closed similar to the issue being experienced.
+-   If the problem can't be resolved, open an issue on `code.availity.com` under the `availity-support` project so that the Availity team can fix or provide feedback.
```

Comment:
You can remove the references to gitlab here. We can say feel free to open an issue. That way if its private or public facing user they know just to create one in general